<launch>
  <node pkg="paul_alarm" type="alarm_omni.py" name="alarm_omni" output="screen"/>
  <!-- <node pkg="paul_alarm" type="dual_image_subscriber_node.py" name="alarm_display" output="screen"/> -->
</launch>
